### PR TITLE
updating codeowners for ff and db devops

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ updating codeowners for ff and db devops
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
     "test": "npm run lint:ci"
   },
   "dependencies": {
-    "@docusaurus/core": "^3.6.2",
+    "@docusaurus/core": "^3.6.3",
```
Comment (pratmit) [inline package.json:22]: Why are these changes in the json file?

Comment (pratmit) [reply package.json:22]: @wei-harness can you confirm on the above?

File: yarn.lock
Comment (wei-harness) [inline yarn.lock:1]: @jenniferopal please don't commit the .lock files

File: package.json (hunk 1 above)
Comment (wei-harness) [reply package.json:22]: same question, @jenniferopal,
We'll handle the dependencies in a more cautious manner,
If there are any features you require from the newer version of Docusaurus, please let us know.

**Review by wei-harness — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (jenniferopal) [reply yarn.lock:1]: My apologies. I didn't realise my updates also merged with the branches. I'll fix that.

**Review by pratmit — APPROVED** [review]

**Review by wei-harness — APPROVED** [review]

**Pull request merged**